src/domain-registry-essentials/cns-smart-contracts.md

```diff
@@ -1,22 +1,22 @@
-# CNS smart contracts
+# CNS Smart Contracts
 
-CNS is built on Ethereum and comprises a bundle of Solidity smart contracts. Their source code is hosted in the [dot-crypto repository](https://github.com/unstoppabledomains/dot-crypto) and maintained by the Unstoppable Domains team. This article lists all the smart contracts from the repository and gives a brief description of each of them, along with the links to the source code and deployment addresses. If you are interested in high-level details about how CNS works, see [Architecture overview](architecture-overview.md).
+Crypto Name Service (CNS) is built on Ethereum and is effectively a bundle of Solidity smart contracts. Their source code is hosted in the [dot-crypto repository](https://github.com/unstoppabledomains/dot-crypto) and maintained by the Unstoppable Domains team. This page lists the smart contracts from that repository and gives a brief description of each along links to the source code and deployment addresses. For high-level details about how CNS works, see [Architecture Overview](architecture-overview.md).
```

Comment:
> and gives a brief description of each along links

It seems to me there is "with" missing after "along". I'm having difficulties with understanding the grammar without it :D